helper.php

```diff
@@ -134,5 +160,73 @@ public function hasUserAcknowledged($page, $user)
 
         return $acktime ? (int)$acktime : false;
     }
+
+    /**
+     * Save user's acknowledgement for a given page
+     *
+     * @param string $page
+     * @param string $user
+     * @return bool
+     */
+    public function saveAcknowledgement($page, $user)
+    {
+        $sqlite = $this->getDB();
+        if (!$sqlite) return false;
+
+        $sql = "REPLACE INTO acks (page, user, ack) VALUES (?,?, strftime('%s','now'))";
```

Comment:
Hmm, shouldn't we store integer timestamps?

Reply:
If I am not mistaken, `strftime('%s','now')` is how SQLite generates the current "integer" timestamp, where `%s` stands for seconds since 01-01-1970.